samples/sensor/bme680/boards/adafruit_feather_nrf52840.overlay

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2020, Patrick Moffitt <patrick@moffitt.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * i2c interface for the Feather nRF52840 Express and the
+ * Adafruit BME680.
+ */
+
+&i2c0 {
+    compatible = "nordic,nrf-twi";
```

Comment:
nit: please replace the leading spaces with a tab

Reply:
Done.